docs/content/usage/integrations/crossplane.md

```diff
@@ -0,0 +1,411 @@
+            metadata:
+              name: {{ $objName }}
+              namespace: default
+            status:
+              ready: {{ and (eq (.observed.resources.database | getResourceCondition "Synced").Status "True") (eq (.observed.resources.user | getResourceCondition "Synced").Status "True") }}
+              connectionSecret: {{ printf "%s-connection-secret" $dbName }}
+            ---
+            apiVersion: mysql.sql.m.crossplane.io/v1alpha1
+            kind: ProviderConfig
+            metadata:
+              name: mysql-cfg
+              annotations:
+                gotemplating.fn.crossplane.io/composition-resource-name: provider-cfg
+                gotemplating.fn.crossplane.io/ready: "True"
+            spec:
+              credentials:
+                source: MySQLConnectionSecret
+                connectionSecretRef:
+                  name: db-conn
+              tls: preferred
+EOF
+```
+
+5. **Export the database API using kube-bind.**
+   Create an APIServiceExportTemplate for the mysqldatabase.mangodb.com resource:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: kube-bind.io/v1alpha2
+kind: APIServiceExportTemplate
+metadata:
+  name: mysqldatabase
+spec:
+  resources:
+    - group: mangodb.com
+      resource: mysqldatabases
+      versions:
+        - v1
+  permissionClaims:
+  - group: ""
+    resource: secrets
+    selector:
+      labelSelector:
+        matchLabels:
+          kube-bind.io/selector: consumer-database
+  scope: Namespaced
+EOF
+```
+
+6. **Login to kube-bind and request a binding to the exported database API.**
+
+    ```bash
+    kubectl bind login https://kube-bind.example.com
+    # Authenticate and select the mysqldatabase export
+    kubectl bind
+    ```
+
+
+7. **Wait for the binding to be established.** Once the binding is active, you can create `MySQLDatabase` resources in your consumer cluster, and you will get `MySQLDatabase` objects synced from the provider cluster.
+
+```bash
+kubectl bind
+🌐 Opening kube-bind UI in your browser...
+    https://kube-bind.example.com?redirect_url=....
+
+Browser opened successfully
+Waiting for binding completion from UI...
+   (Press Ctrl+C to cancel)
+
+Binding completed successfully!
+🔒 Updated secret kube-bind/kubeconfig-zxrdn for host https://kube-bind.example.com, namespace kube-bind-bp52k
+🚀 Deploying konnector v0.6.0 to namespace kube-bind with custom image "ghcr.io/kube-bind/konnector:v0.6.0".
+✅ Created APIServiceBinding mysqldatabase-6rvjt for 1 resources
+Created 1 APIServiceBinding(s):
+  - mysqldatabase-6rvjt
+Resources bound successfully!
+```
+
+8. **Create a managed database in your consumer cluster.**
+    Verify that mysqldatabase.mongodb.com CRD is synced to the consumer cluster:
+```bash
+k get crd mysqldatabases.mangodb.com
+NAME                         CREATED AT
+mysqldatabases.mangodb.com   2025-11-27T14:22:18Z
+```
+    Order a new consumer-database instance in the provider cluster
+
+```yaml
+kubectl apply -f - <<EOF
```

Comment:
I get :

```
- lastTransitionTime: "2025-12-11T08:31:15Z"
      message: 'cannot compose resources: cannot run Composition pipeline step "create-mysql-resources":
        cannot get gRPC client connection for Function "function-go-templating": cannot
        find an active FunctionRevision (a FunctionRevision with spec.desiredState:
        Active)'
```

feels like composition is wrong :/

Reply:
or functions are missing...

Reply:
Next I get :
```
    - lastTransitionTime: "2025-12-11T08:36:49Z"
      message: 'connect failed: cannot get credentials Secret: Secret "db-conn" not
        found'
      observedGeneration: 1
      reason: ReconcileError
      status: "False"
      type: Synced
```

feels like this doc is not updated

Reply:
Sorry, was focused on testing the permission claims too much, did not copy over the configuration properly from that remote repo